droolsjbpm-brms-distribution/pom.xml

```diff
@@ -77,6 +83,12 @@
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>kie-karaf-features</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.activation</groupId>
```

Comment:
same as above... since version 2. it was moved to jakarta.activation
this depends on our artifacts so the exclusion should match